dice.js

```diff
@@ -0,0 +1,13 @@
+function rollDie(sides)
+  {
+    if(!sides) sides = 6;
+    with(Math) return 1 + floor(random() * sides);
```

Comment:
While this is correct, `with` is not really recommended. 
You can simply avoid it by calling `Math.floor(Math.random()*sides)`

From the MDN documentation: 
>Use of the with statement is not recommended, as it may be the source of confusing bugs and compatibility issues.